doc/nrf/ug_thread_architectures.rst

```diff
@@ -0,0 +1,160 @@
+.. _ug_thread_architectures:
+
+OpenThread architectures
+########################
+
+This page describes the OpenThread stack architecture and platform designs that are possible with the OpenThread network stack on Nordic Semiconductor devices in |NCS|.
+
+The designs are described from the least to the most complex, that is from simple applications that consist of a single chip running single or multiple protocols to scenarios in which the nRF SoC acts as a network co-processor when the application is running on a much more powerful host processor.
+
+.. contents::
+    :local:
+    :depth: 1
+
+The platform designs work with the following development kits:
```

Comment:
I'm not sure if I get it right, but did you intentionally mentioned only DKs? As far as I know RCP main target is nrf52811, but there's not DK with it.

Reply:
From what Lukasz Duda told me, RCP is not yet supported, hence no mention of 811. There's also a note about missing support in the RCP section.